### ISSUE TITLE
update script error: /usr/local/bin/docker-compose: line 1: Not: command not found

### ISSUE THREAD

**Issue by unze** [mailcow/mailcow-dockerized#4280]
updated like usual, getting following error:

```
Fetching new images, if any...
/usr/local/bin/docker-compose: line 1: Not: command not found
Checking IPv6 settings...
Starting mailcow...
/usr/local/bin/docker-compose: line 1: Not: command not found
Collecting garbage...
Further cleanup...
If you want to cleanup further garbage collected by Docker, please make sure all containers are up and running before cleaning your system by executing "docker system prune"
root@mail ~ # ./update-mailcow
Checking internet connection... OK
Checking for newer update script...
Updated 0 paths from b683fee3
Are you sure you want to update mailcow: dockerized? All containers will be stopped. [y/N] y
Great! Native IPv6 NAT is active.
Validating docker-compose stack configuration...
/usr/local/bin/docker-compose: line 1: Not: command not found

Oh no, something went wrong. Please check the error message above.
```

content of /usr/local/bin/docker-compose:
`Not found`

**Comment by dofl** [comment]
Docker-compose 2.0 has just been release into the wild which breaks things. Downgrade your docker-compose until we have a fix.

```
sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose

chmod +x /usr/local/bin/docker-compose
```

Beware, the update script downloads and installs the new docker-compose as well. So you need to manually start the dockers after running the update script. 

`docker-compose up -d --remove-orphans`

**Comment by andryyy** [comment]
I implemented a quick workaround/"fix" to return 1.29.2 until we migrated.
